packages/circuit-ui/components/Modal/Modal.mdx

```diff
@@ -5,58 +5,116 @@ import * as Stories from './Modal.stories';
 
 # Modal
 
-<Status variant="under-review" />
+<Status variant="stable" />
 
-The modal component displays self-contained tasks in a focused window that overlays the page content.
+The modal component displays self-contained tasks in an overlay view, requiring the user to interact with it before returning to the underlying content.
 
 <Story of={Stories.Base} />
 <Props />
 
 ## When to use it
 
-Generally, use the modal component sparingly. Consider displaying more complex tasks and large amounts of information on a separate page instead.
+Generally, use the modal dialog component sparingly. Consider displaying more complex tasks and large amounts of information on a separate page instead.
 
-## Variants
+A modal dialog is a disruptive pattern that interrupts the user's current task. It should only be used when the task requires immediate attention or when the user needs to make a decision before continuing:
 
-<Story of={Stories.Variants} />
+- Confirming a critical action, such as deleting an item or abandoning a task (also see the [NotificationModal](Notification/NotificationModal/Docs) component).
+- Forms or inputs to collect small amounts of data without navigating away from the current page (e.g., login, feedback forms).
+- Focused tasks like editing an item, uploading files, or reviewing details.
 
-### Contextual
```

Comment:
What happened to the contextual variant (docs)?

Reply:
Even though it's the default variant, we need to document the `contextual` variant to help users choose between the variants.